oia/armando-moviles/src/armadomoviles/ArmadoMoviles.java

```diff
@@ -0,0 +1,23 @@
+
+package armadomoviles;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+
+public class ArmadoMoviles {
+
+    
+    public static void main(String[] args) throws FileNotFoundException, IOException {
+    
+        Archivo ar = new Archivo("colgantes.in");
```

Comment:
Acá veo por qué usamos el setter... creo que sería más fácil que se inicialice con "colgantes", y dentro sepa que si está leyendo, es `.in`, y si está escribiendo, es `.out`.
¿Qué opinan?